<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompareParamsPage</class>
 <widget class="QWizardPage" name="CompareParamsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>133</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radioText">
         <property name="text">
          <string>Со структурой с++</string>
         </property>
         <property name="icon">
          <iconset resource="../../../res/fmt.qrc">
           <normaloff>:/cplusplus</normaloff>:/cplusplus</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioTable">
         <property name="text">
          <string>С другой таблицей схемы</string>
         </property>
         <property name="icon">
          <iconset resource="../../../res/fmt.qrc">
           <normaloff>:/table</normaloff>:/table</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioConnection">
         <property name="text">
          <string>С записью fmt из другой бд</string>
         </property>
         <property name="icon">
          <iconset resource="../../../res/fmt.qrc">
           <normaloff>:/db</normaloff>:/db</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioXml">
         <property name="text">
          <string>С записью fmt из другой бд к которой нет подключения</string>
         </property>
         <property name="icon">
          <iconset resource="../../../res/fmt.qrc">
           <normaloff>:/img/base_cylinder_Gray.png</normaloff>:/img/base_cylinder_Gray.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>318</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../res/fmt.qrc"/>
 </resources>
 <connections/>
</ui>
